### PR TITLE
Contributing Guidelines for RIF Relay

### PR DESCRIPTION
**Pull request by zdaodu-iov**
## What

- Created a Contributing Guidelines for RIF Relay

## Why

- It helps maintain consistency in code, documentation, and other contributions. 
- It sets a standard for quality, ensuring that all contributions meet the project's requirements.

## Refs

- (optional: include links to other issues, PRs, tickets, etc)

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Rootstock Contribution Guidelines - RIF Relay
+Thank you for investing your time in contributing to RIF Relay! Any contribution you make will be reflected on the [RIF Relay GitHub page](https://github.com/rsksmart/rif-relay). Your efforts help improve the Rootstock and RIF ecosystem.
+
+## Understanding the Project
+RIF Relay is a secure transaction relay system that enables users to pay fees using ERC-20 tokens; it was inspired by the Gas Station Network (GSN) project. It comprises four main modules: RIF Relay Contracts, RIF Relay Common, RIF Relay Client, and RIF Relay Server, each with specific functionalities and roles within the system.
```
Comment (antomor) [inline contributing.md:5]: RIF Relay common has been removed. Furthermore, I'd also add the link to their respective repositories.
```suggestion
RIF Relay is a secure transaction relay system that enables users to pay fees using ERC-20 tokens; it was inspired by the Gas Station Network (GSN) project. It comprises four main modules: RIF Relay Contracts, RIF Relay Client, and RIF Relay Server, each with specific functionalities and roles within the system.
```

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+2. Create a new branch for your fix.
+3. Make your changes in the new branch.
+4. Commit your changes with a descriptive commit message.
+5. Push the branch to your fork.
+6. Submit a pull request against the main branch of the original repository.
+
+**Proposing New Features**
+1. Open an issue on the GitHub repository describing the feature.
+2. Discuss the feasibility and relevance of the feature with the project maintainers.
+3. Once approved, follow the steps for submitting a fix to implement the feature.
```
Comment (antomor) [inline contributing.md:36]:
```suggestion
3. Once approved, follow the steps to submit the changes required to implement the feature.
```

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+**How to Submit an Issue**
```
Comment (antomor) [inline contributing.md:38]: What's the difference between issue and fix?

File: contributing.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+**How to Submit a PR**
+1. Follow the same steps as submitting an issue, ensuring your changes are in a separate branch.
+2. Provide a clear and detailed description of the changes in the PR.
+
+**Testing**
+- Follow the testing instructions in the [Readme.md](https://github.com/rsksmart/rif-relay#readme) to run tests.
+- Ensure all tests pass before submitting a PR.
+
+**Where to go for help**
```
Comment (antomor) [inline contributing.md:54]: This part is repeated in the "New Contributor" section

**Review by antomor — CHANGES_REQUESTED** [review]
@zdaodu-iov thank you for the changes. I've left some minor comments

File: contributing.md (hunk 3 above)
Comment (zdaodu-iov) [reply contributing.md:38]: @antomor I followed the template that was given to me by Owanate.
But I think An issue is a report about a bug, enhancement, or other task related to the project. It's essentially a way to track work that needs to be done.

A fix is the actual code or documentation change that resolves an issue. This is usually done through a commit or a series of commits in the project's repository.

Comment (antomor) [reply contributing.md:38]: @zdaodu-iov I see that the theoretical difference, but I don't get what's the difference in the described process. For what I see here, the process looks the same:
1. fork the repo
2. create a new branch
3. push the changes
4. create a PR

Isn't the process the same? Why do we need to have 2 sections? Sorry if I'm missing something

Comment (zdaodu-iov) [reply contributing.md:38]: Some people tend to get confused with it but then we can remove it if you find it less important

Comment (antomor) [reply contributing.md:38]: Yes, please, I don't like repetitions, because usually, at some point, they get out of sync 😅

Comment (zdaodu-iov) [reply contributing.md]: Okay, edited it

**Review by antomor — APPROVED** [review]

**Review by owans — APPROVED** [review]

**Review by antomor — APPROVED** [review]

**Pull request merged**